docs/csharp/language-reference/keywords/out.md

```diff
@@ -18,6 +18,36 @@ You can use the `out` keyword in two contexts:
 
 - In [generic type parameter declarations](out-generic-modifier.md) for interfaces and delegates, which specifies that a type parameter is covariant.
 
+## Example
+The out keyword can be especially useful when you want to get 2 values back from a method for 2 local variables, without using a custom object, tuple or class-scoped variable. In the below example, the method `IsDivisibleBy2` takes in an integer, determines if it is divisible by 2, returns the result as a `bool` and outs the division result.
+
+   
+    public static bool IsDivisibleBy2(int x, out double dividedBy2)
```

Comment:
There's a markdown lint warning for the style (indent vs. triple-backtick). Instead, you should add the sample to the associated snippets project, so the new code is validated in our CI system.

Reply:
Where can I find the associated snippets project?

Reply:
I think there's none currently, but you could add one somewhere under https://github.com/dotnet/docs/tree/main/samples/snippets/csharp/language-reference/keywords